src/Annotations/Serializer/HypothesisClientAnnotationNormalizer.php

```diff
@@ -80,7 +80,7 @@ public function normalize($object, $format = null, array $context = []) : array
 
     private function processText(string $text) : array
     {
-        $blocks = $this->docParser->parse($text)->children();
+        $blocks = $this->docParser->parse(escape_math($text))->children();
```

Comment:
This means we're escaping Math elements before, then purifying after? This could also compose `HtmlRenderer`?

(Means more regexes, but we're going to be better off caching the whole thing anyway.)

Reply:
The Math elements are escaped while the content is in Markdown. We need to determine what `CommonMark` `Block` elements that we have before rendering the HTML.

Reply:
You mean that `CommonMark` does something with HTML element? Assumed it wouldn't...